src/bp/core/repositories/events.ts

```diff
@@ -54,6 +55,13 @@ export class KnexEventRepository implements EventRepository {
     )
   }
 
+  async putEvent(id: number, fields): Promise<void> {
+    await this.database
+      .knex(this.TABLE_NAME)
+      .where({ id })
+      .update(fields)
+  }
```

Comment:
I think updateEvent would be a better naming, because I expect an insert with put